applications/orochi-network-orosign-part1.md

```diff
@@ -0,0 +1,169 @@
+| **0c.** | Testing and Testing Guide             | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                                   |
+| **0d.** | Docker                                | We will publish an article that explains about Orosign's MPC wallet on curve `secp256k1`                                                                                                                                                          |
+| **1.**  | Functionality: MPC on secp256k1 curve | Implement MPC for `secp256k1`, supporting multiparty computation to perform transaction/message signing with `secp256k1`. This module will implement with Rust programing language and compile to Wasm opcode or native lib for React Native app. |
+
+### Milestone 3
+
+- **Estimated Duration:** 2 months
+- **FTE:** 1.6
+- **Costs:** $11,200
+
+| Number  | Deliverable                         | Specification                                                                                                                                                                                                                                 |
+| ------- | ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License                             | Apache 2.0                                                                                                                                                                                                                                    |
+| **0b.** | Documentation                       | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works.         |
+| **0c.** | Testing and Testing Guide           | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                               |
+| **0d.** | Docker                              | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                                 |
```

Comment:
Is Docker needed for this milestone? If you don't think it is necessary you can go ahead and remove it.